diff --git a/applets/systemloadviewer/systemloadviewer.cpp b/applets/systemloadviewer/systemloadviewer.cpp
index bff0b34..2da353d 100644
--- a/applets/systemloadviewer/systemloadviewer.cpp
+++ b/applets/systemloadviewer/systemloadviewer.cpp
@@ -468,7 +468,7 @@ void SystemLoadViewer::dataUpdated(const QString& source, const Plasma::DataEngi
         }
 
         m_swaptotal = m_swapfree + m_swapused;
-        if(qFuzzyCompare(m_swaptotal+1, 1.0)) {
+        if(qFuzzyCompare(m_swaptotal+1, qreal(1.0))) {
             if(m_swapAvailable) {
                 m_swapAvailable = false;
                 updateConstraints(Plasma::SizeConstraint);
diff --git a/applets/timer/timer.cpp b/applets/timer/timer.cpp
index ba5ee66..6fe63b1 100644
--- a/applets/timer/timer.cpp
+++ b/applets/timer/timer.cpp
@@ -498,7 +498,7 @@ void Timer::toggleTimerVisible()
         m_blinkAnim = animGroup;
     }
 
-    m_blinkAnim->setDirection(qFuzzyCompare(m_hoursDigit[0]->opacity(), 1.0) ?
+    m_blinkAnim->setDirection(qFuzzyCompare(m_hoursDigit[0]->opacity(), qreal(1.0)) ?
                               QAbstractAnimation::Forward : QAbstractAnimation::Backward);
     m_blinkAnim->start();
 }
